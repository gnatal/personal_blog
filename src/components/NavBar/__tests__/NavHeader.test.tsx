import React from 'react';
import renderer from 'react-test-renderer';
import NavHeader from '../Navheader';

test('It should write an span in the nav header', () => {
  const component = renderer.create(<NavHeader blogName='Natal ideias' />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
