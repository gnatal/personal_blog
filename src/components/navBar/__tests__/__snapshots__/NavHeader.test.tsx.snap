// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should write an span in the nav header 1`] = `
<div
  className="flex items-center flex-shrink-0 text-white mr-6"
>
  <span
    className="font-semibold text-xl tracking-tight"
  >
    Natal ideias
  </span>
</div>
`;
